andela/ah-frontend-athena
# #162163224 Setup project to use sass

### What does this PR do?
This Pull Request sets up the project to use sass.

### Description of Task to be completed?
This involved installing node-sass to enable the project to interpret sass.
All the css files were changed to scss

### How should this be manually tested?
Clone the repo
Check out the ch-setup-redux-162163220 branch
run npm install
run npm start to start the development server.
The project is able to run with scss files in it which shows it can compile scss.

### What are the relevant pivotal tracker stories?
#162163224
